scripts/gulpfiles/build_tasks.js

```diff
@@ -559,7 +563,10 @@ function getChunkOptions() {
     // Figure out which chunk this is by looking for one of the
     // known chunk entrypoints in chunkFiles.  N.B.: O(n*m).  :-(
     const chunk = chunks.find(
-        chunk => chunkFiles.find(f => f.endsWith(path.sep + chunk.entry)));
+        chunk => chunkFiles.find(f => {
+          return f.endsWith(path.sep.replace("\\", "/") + chunk.entry.replaceAll("\\", "/"));
+        }
+        ));
```

Comment:
What I'm unclear about—and what would be very helpful if you could give us some information about—is what the strings in the `chunkFiles` array look like.

On macOS (and Linux etc.) they use `/` as a separator, and I was expecting that they would use `\` on win32, and so using `f.endsWith(path.sep + chunk.entry)` (where `chunk.entry` also uses `path.sep`) should work fine.

But from this proposed code change it seems that for `chunkFiles` uses `/` as a path separator even on Windows.  Is that correct?

In any case, the proposed change can be simplified somewhat, since `path.sep` is only ever either `'/'` or `'\\'`:

```suggestion
        chunk => chunkFiles.find(f => f.endsWith('/' + chunk.entry.replaceAll("\\", "/")));
```

Reply:
Windows accepts both `\` and `/` which is why, in my projects, I generally opt in for `path.replaceAll("\\", "/")` to make the behavior consistent across os systems. The reason I also did not replace `path.sep` is because I did not want to introduce extra changes. I wanted to keep the original code the same as much as possible.

There are all chunk entries that pass the condition (f.endsWith):
```
build\src\core\main.js
build\src\blocks\blocks.js
build\src\generators\javascript.js
build\src\generators\python.js
build\src\generators\php.js
build\src\generators\lua.js
build\src\generators\dart.js
```